staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -362,8 +362,10 @@ func (e *Store) WaitForInitialized(ctx genericapirequest.Context, obj runtime.Ob
 		select {
 		case event, ok := <-ch:
 			if !ok {
-				// TODO: should we just expose the partially initialized object?
-				return nil, kubeerr.NewServerTimeout(e.QualifiedResource, "create", 0)
+				return latest,
```

Comment:
The current code is unlikely to hit this path - you're going to get server timeout first (which is controlled at a much higher level).  

If you get a server timeout on a CREATE, why isn't that sufficient to know that the request may still be continuing?  Why do you need a special error?  After all, ServerTimeout already says "the operation may still be in progress" (or if it doesn't maybe we're using the wrong timeout there.